tests/fhir/test_fhir_client.py

```diff
@@ -227,7 +227,7 @@ async def test_bad_smart_config(self, bad_config_override):
             {},
             (
                 "Client error '400 Bad Request' for url 'https://auth.example.com/token'\n"
-                "For more information check: https://httpstatuses.com/400"
+                "For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400"
```

Comment:
Unrelated, but httpx changed its output format. Maybe we should be more resilient to that, but part of me likes the strictness and doesn't mind updating once in a blue moon.

Reply:
honestly i don't hate it either, and that's probably a better source for that info anyway.